src/main/java/seedu/duke/manager/OrderManager.java

```diff
@@ -1,46 +1,70 @@
 package seedu.duke.manager;
 
+import seedu.duke.controllers.DishController;
 import seedu.duke.entities.Dish;
 import seedu.duke.entities.Order;
 
 import java.util.ArrayList;
 
-
+/**
+ * OrderManager.
+ */
 public class OrderManager {
-    private final ArrayList<Order> orders;
-    private final DishManager dishManager = new DishManager();
-    private final ArrayList<Dish> dishes;
+    private ArrayList<Order> orders;
+    private ArrayList<Dish> dishes;
 
-    public OrderManager() {
+    public OrderManager(DishController dishController) {
```

Comment:
this will change once we change the managers to have singleton pattern right?

Reply:
ah I alr replaced all `new DishManager()` with `DishManager.getInstance()`

fetch and solve the conflict and this should be fine

Reply:
ok then will let ziyan settle

Reply:
what should I do? fetch and change what